src/workerd/jsg/README.md

```diff
@@ -886,6 +901,12 @@ public:
 };
 ```
 
+// r: it would be super nice, if the read values of 'hello' and 123 were set somewhere in the C++
+//    code, or were passed into the constructor in JS (but I think the former is maybe preferable here?);
+//    like that they would feel a little more tangible, the sample would be a little more complete,
+//    even though the user still has to imagine the getter and setter implementations
+//    (my C++-fu as well as my JSG understanding is too weak, as of yet, to confidently suggest
+//    the necessary changes)
```

Comment:
It would be simply...

```cpp
class Foo: public jsg::Object {
public:
  static jsg::Ref<Foo> constructor();

  kj::String getAbc() { return kj::str("hello"); }
  int getXyz() { return xyz; }
  void setXyz(int value) { xyz = value; }

  JSG_RESOURCE_TYPE(Foo) {
    JSG_READONLY_PROTOTYPE_PROPERTY(abc, getAbc);
    JSG_PROTOTYPE_PROPERTY(xyz, getXyz, setXyz);
  }

private:
  int xyz = 123;
};
```